script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -857,7 +857,7 @@ Once I've cut off his supply of blood,
 -- Page 395, Offset 17559.
 -- 弱っているのを隠したところで、そんな体じゃ逆に殺されるようなものなんだろ……！　なら―――」
 Even if you hide the fact that you're weak right now,
- with your body in that state you're just going to get yourself killed...!
+ with your body in that state, you're just going to get yourself killed...!
```

Comment:
not sure this one makes much sense either but I'm actually not confident in that judgement